test/tests/functional/pbs_checkpoint.py

```diff
@@ -44,40 +44,67 @@ class TestCheckpoint(TestFunctional):
     """
     This test suite targets Checkpoint functionality.
     """
+    abort_file = ''
+    cu = CrayUtils()
 
-    def test_checkpoint_abort_with_preemept(self):
+    def test_checkpoint_abort_with_preempt(self):
+        """
+        This test verifies that checkpoint_abort works as expected by
+        1. setting preempt_order in sched_config to 'C'.
+        2. submitting a job to the default queue.
+        3. submitting a job to the express queue.
+        4. confirming that the first job was checkpointed.
```

Comment:
Don't put the test steps in the doc string.  Just describe the test.

Reply:
ok, done!